### DISCUSSION TITLE
How do I verify a `SecretKey` signature on jwt.io ?

### DISCUSSION THREAD

**Discussion by emt2dev** [jwtk/jjwt#860] [Q&A] [answered]
**Describe the bug**
I've tried to create and parse a jwt using spring boot api and jwt.io
eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhdHRkc2FkYWRzZHRAdXNlci5jb20ifQ.GEiJSiu6EdSHKexspsHFOjmo9dwH_lV94KWrZLiZOQs
shows as  invalid signature on jwt.io

**To Reproduce**

I've kept it very simple:
SecretKey key = Jwts.SIG.HS256.key().build();
return Jwts.builder().subject(userDetails.getUsername()).signWith(key).compact();

**Expected behavior**
I'd like to have a valid jwt.

**Screenshots**
![image](https://github.com/jwtk/jjwt/assets/86013234/72ce66f1-5ae6-4fc6-8e8a-f8fc5d4d680e)

**Comment by laurids** [comment, marked as answer]
You need to let jwt.io know the key in order for it to verify the signature.
Just paste the base64 encoded key into the text box on the bottom right saying "your-256-bit-secret".
`java.util.Base64.getEncoder().encodeToString(key.getEncoded());`
